### PR TITLE
FAQ, jobs, CF updates

### PR DESCRIPTION
**Pull request by dsajdak**

### CODE REVIEW

File: pages/getting-access.md [hunk 1]
```diff
@@ -79,3 +79,51 @@ allocation.
   request a CCR account and VPN access by [contacting CCR Help](help.md)
 
 - __Industry customers__ please [contact CCR Help](help.md) to request access.  
+
+## Available Resources
```
Comment (aebruno) [inline pages/getting-access.md:83]: This all looks great. What about making this it's own page though? I think having it as a first class menu option on the left side would be great?

File: pages/portals/coldfront.md [hunk 2]
```diff
@@ -41,6 +41,9 @@ project has been created take a moment to add any publications, grants, or
 research output which helps us keep track of the scientific impact of our
 center. You can also [add users](#add-or-remove-users) to your project.
 
+!!! Note "New Faculty Must Request PI Status"
```
Comment (aebruno) [inline pages/portals/coldfront.md:44]: We also have this same exact thing [here](https://docs.ccr.buffalo.edu/en/latest/getting-access/#enable-two-factor-authentication) but worded differently. I'm not sure it's worth repeating again here but if you think it is, let's use the exact same wording?

**Review by aebruno — CHANGES_REQUESTED** [review]
Looks great. Two minor suggestions

File: pages/getting-access.md (hunk 1 above)
Comment (dsajdak) [reply pages/getting-access.md]: Moved to separate page and added some additional info

File: pages/portals/coldfront.md (hunk 2 above)
Comment (dsajdak) [reply pages/portals/coldfront.md]: I did not see this!  I was wondering why we didn't have it somewhere so I added it.  I've removed this though as I agree we don't need to repeat it.

**Review by aebruno — APPROVED** [review]

**Pull request merged**